### PR TITLE
Cursor will now move to the end after evaluation in calculator activity

### PR DESCRIPTION
**Pull request by Scar26**
Hopefully this solves issue #318 
Please review @llaske

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,10 +30,15 @@
       "category": "public.app-category.education"
     },
     "win": {
-      "target": [ "nsis" ]
+      "target": [
```
Comment (llaske) [inline package.json]: Change not related with this PR.

File: package.json [hunk 2]
```diff
@@ -30,10 +30,15 @@
...
+        "nsis"
+      ]
     },
     "linux": {
-      "target": [ "AppImage", "deb" ]
+      "target": [
```
Comment (llaske) [inline package.json]: Change not related with this PR.

**Comment by Scar26** [comment]
Done @llaske

File: package.json (hunk 2 above)
Comment (Scar26) [reply package.json]: reverted unnecessary changes

**Comment by Scar26** [comment]
Is there anything else I should change with respect to this PR @llaske ?

**Comment by llaske** [comment]
An easiest solution here: https://github.com/llaske/sugarizer/pull/321

**Pull request closed without merging**